pkg/products/threescale/envoyconfig.go

```diff
@@ -106,6 +108,7 @@ var multiTenantRatelimitDescriptor = envoyroutev3.RateLimit{
 /**
  httpFilters:
 	- &tsHTTPRateLimitFilter
+	- name: envoy.filters.http.grpc_http1_reverse_bridge
```

Comment:
Can I ask why we include commented out sections?

Reply:
Can't say I know the answer. These were already present from Jose's initial work and was just continued for consistency 🤷

Reply:
I'm happy to keep it is for now. We can create a ticket for removing / improving the comments if we feel strongly about it

Reply:
Thanks for the info guys. No need to remove these urgently. Just wondered what the reason was to have them. If they contain helpful information then no objection to keeping them.